crates/macro/src/html_tree/html_tag/tag_attributes.rs

```diff
@@ -24,6 +24,53 @@ pub enum ClassesForm {
     Single(Expr),
 }
 
+lazy_static! {
```

Comment:
Can you please add a comment saying what the source of this list is?

Reply:
Looks like this list is not totally correct, see this comment: https://gist.github.com/ArjanSchouten/0b8574a6ad7f5065a5e7#gistcomment-2717606